test/plots/rectanglePlotTests.ts

```diff
@@ -2,444 +2,418 @@
 
 describe("Plots", () => {
   describe("RectanglePlot", () => {
-    let SVG_WIDTH = 300;
-    let SVG_HEIGHT = 300;
-    let DATA = [
-      { x: 0, y: 0, x2: 1, y2: 1 },
-      { x: 1, y: 1, x2: 2, y2: 2 },
-      { x: 2, y: 2, x2: 3, y2: 3 },
-      { x: 3, y: 3, x2: 4, y2: 4 },
-      { x: 4, y: 4, x2: 5, y2: 5 }
-    ];
-    let VERIFY_CELLS = (cells: d3.Selection<any>) => {
-      assert.strictEqual(cells[0].length, 5);
-      cells.each(function(d: any, i: number) {
-        let cell = d3.select(this);
-        assert.closeTo(+cell.attr("height"), 50, 0.5, "Cell height is correct");
-        assert.closeTo(+cell.attr("width"), 50, 0.5, "Cell width is correct");
-        assert.closeTo(+cell.attr("x"), 25 + 50 * i, 0.5, "Cell x coordinate is correct");
-        assert.closeTo(+cell.attr("y"), 25 + 50 * (cells[0].length - i - 1), 0.5, "Cell y coordinate is correct");
+    describe("RectanglePlot - basics", () => {
```

Comment:
It would be best if we can narrow down what "general usage" and "basics" mean...

Maybe "renders rectangles if x/y/width/height properties are set"?  I think the reason why our testing infrastructure has slowly decayed is like because we chose ambiguous names in the past